custom/icds_reports/models/aggregate.py

```diff
@@ -1587,6 +1589,32 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateAvailingServiceForms(models.Model, AggregateMixin):
+    """ Aggregated data for the availing services
+
+    A availing services exists for each state_id
+
+    A row exists for every person case that has had a record of migration
+    submitted against it this month
+    """
+    state_id = models.CharField(max_length=10)
+    supervisor_id = models.TextField(null=True)
+    awc_id = models.TextField(null=True)
+    month = models.DateField(help_text="Will always be YYYY-MM-01")
+
+    # not the real pkey - see unique_together
+    person_case_id = models.CharField(max_length=40, primary_key=True)
+
+    is_registered = models.PositiveSmallIntegerField(blank=True, null=True, help_text="Status of the Registration")
+
+    class Meta(object):
+        db_table = AGG_AVAILING_SERVICES_TABLE
+        unique_together = ('month', 'supervisor_id', 'person_case_id')  # pkey
+
+    _agg_helper_cls = AvailingServiceFormsAggregationDistributedHelper
```

Comment:
Can you also add its agg function to tasks files and call it in move_ucr function like migration one. So that we can ensure its at least aggregating and not failing.